cwms-data-api/src/main/java/cwms/cda/data/dto/timeseriesprofile/TimeSeriesProfileParser.java

```diff
@@ -0,0 +1,194 @@
+package cwms.cda.data.dto.timeseriesprofile;
+
+import java.math.BigInteger;
+import java.util.List;
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import cwms.cda.api.errors.FieldException;
+import cwms.cda.data.dto.CwmsDTO;
+import cwms.cda.formatters.Formats;
+import cwms.cda.formatters.annotations.FormattableWith;
+import cwms.cda.formatters.json.JsonV2;
+
+@FormattableWith(contentType = Formats.JSONV2, formatter = JsonV2.class)
+@JsonDeserialize(builder = TimeSeriesProfileParser.Builder.class)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
+public class TimeSeriesProfileParser extends CwmsDTO
+{
+	private final String locationId;
+	private final String keyParameter;
```

Comment:
re-use ParameterInfo?

Reply:
note the DB has separate tables for the profile params vs profile parser params

Reply:
Here are the DB fields, 
LOCATION_CODE,
KEY_PARAMETER_CODE,
TIME_FIELD, 
TIME_IN_TWO_FIELDS, 
TIME_COL_START, 
TIME_COL_END, 
TIME_ZONE_CODE, 
TIME_FORMAT, 
RECORD_DELIMITER, 
FIELD_DELIMITER

Reply:
Do you need to include time col start/end?

Reply:
keyParameter is just the name of the keyParameter, its ParameterInfo is part of the parameterInfoList
added colStart and colEnd as part of TimeSeriesProfileParserColumnar